### DISCUSSION TITLE
I was wondering if there were spaces where people shared their prompts, their tutorials, their general discussion around the subject of voice cloning and voice generating, particularily related to text-generation-webui?

### DISCUSSION THREAD

**Discussion by AIhasArrived** [rsxdalv/TTS-WebUI#207] [Q&A]
I was wondering if there were spaces where people shared their prompts, their tutorials, their general discussion around the subject of voice cloning and voice generating, particularily related to text-generation-webui?

**Comment by rsxdalv** [comment]
The information is scattered in many places. What's the format you are looking for?

It could be done here, or in README or Youtube or Discord, etc. I also hope to integrate some of the information with the UI to make it more accessible. For example, being able to click on "CFG Scale" to find out what that actually is etc.

**Comment by AIhasArrived** [comment]
I would like to know to know what "every" tab does, with at least one little video or series of screenshot explaining how to do it step by step. for example.
Is there documentation on discord? I was thinking about reddit. Do you mind if I make one?

**Reply by rsxdalv** [reply]
Yeah you can make a reddit. I can't promise that I would call it an "official" reddit, but I think it's great if people make their own communities and media, after all this is open source.
I was thinking about making a discord but didn't do it yet because I haven't decided on the moderation and rules.

**Reply by AIhasArrived** [reply]
Its defintely a must!  I will keep you informed

**Comment by rsxdalv** [comment]
I'll check in tomorrow

**Reply by AIhasArrived** [reply]
Quick question man @rsxdalv : Is there a way to make the audio generated take the exact amount of time you desire? So when I write for instance a sentence of 5 words, I could control whether the ai reader makes it in 5, 6 or 7 seconds?

**Comment by rsxdalv** [comment]
No, but you could always edit it with audacity to change the tempo. It is
possible to get the ai to speak faster or slower but since these are
transformers, they misbehave.

**Reply by AIhasArrived** [reply]
I don't know audacity yet all that much, 
IS changing the temp a one click thing? Or is it complicated thing?
Also can it be automated through code? (and audacity coding?)

**Comment by rsxdalv** [comment]
It can be automated, it's not that hard. As long as it's a small
difference, it might be unnoticeable.
